src/helix-ui/elements/HXAccordionElement.js

```diff
@@ -0,0 +1,103 @@
+import { HXElement } from './HXElement';
+
+export class HXAccordionElement extends HXElement {
+    static get is () {
+        return 'hx-accordion';
+    }
+
+    constructor () {
+        super();
+        this._onPanelOpen = this._onPanelOpen.bind(this);
+    }
+
+    connectedCallback () {
+        this.$upgradeProperty('currentPanel');
+        this.panels.forEach(panel => {
+            panel.addEventListener('open', this._onPanelOpen);
+        });
+    }
+
+    disconnectedCallback () {
+        this.panels.forEach(panel => {
+            panel.removeEventListener('open', this._onPanelOpen);
+        });
+    }
+
+    static get observedAttributes () {
+        return [ 'current-panel' ];
+    }
+
+    attributeChangedCallback (attr, oldVal, newVal) {
+        if (newVal !== null) {
+            this._openPanel(Number(newVal));
+        }
+    }
+
+    // PUBLIC PROPERTIES
+
+    // TODO: needs tweaked for nested accordions (e.g. multi-level navigation)
+    // As it currently is, this will return ALL panels within the accordion,
+    // not just the immediate children.
+    get panels () {
+        return Array.from(this.querySelectorAll('hx-accordion-panel'));
+    }
+
+    get currentPanel () {
+        return Number(this.getAttribute('current-panel'));
+    }
+
+    set currentPanel (idx) {
+        if (idx !== null) {
+            let maxIndex = this.panels.length - 1;
+
+            if (idx >= 0 && idx <= maxIndex) {
+                this.setAttribute('current-panel', idx);
+            }
+        } else {
+            this.removeAttribute('current-panel');
+        }
+    }
+
+    // PUBLIC METHODS
+
+    nextPanel () {
+        if (this._isNavigable) {
+            this.currentPanel += 1;
+        }
+    }
+
+    previousPanel () {
+        if (this._isNavigable) {
+            this.currentPanel -= 1;
+        }
+    }
+
+    // PRIVATE PROPERTIES
+
+    get _isNavigable () {
```

Comment:
Converted from `_isNavigable()` (function) to `_isNavigable` (property) for simplicity.